style.css

```diff
@@ -352,11 +356,69 @@ textarea {
   display: none;
 }
 
-@media only screen and (max-width: 574px){
-  .hamburger{
+@media only screen and (max-width: 574px) {
+  .hamburger {
+    display: block;
+  }
+
+  .hamburger.show .bars:nth-child(2) {
+    opacity: 0;
+  }
+
+  .hamburger.show .bars:nth-child(1) {
+    transform: translateY(8px) rotate(45deg);
+  }
+
+  .hamburger.show .bars:nth-child(3) {
+    transform: translateY(-8px) rotate(-45deg);
+  }
+
+  .navbarlist {
     display: block;
+    position: fixed;
+    left: -100%;
+    top: 1px;
+    height: 555px;
+    background: #191765;
+    background-image: url("IMG1/mb-menu-bg.png");
+    flex-direction: column;
+    text-align: left;
+    transition: 0.3s;
+    width: 100%;
+    z-index: 2;
```

Comment:
- [ ] great job stying the mobile menu however there is a minor change you need to make. the height should take up the entire height of the screen, please fix that.
- [ ] The arrows should be in the same surface with the respective text `Portfolio`, `About`, `Contact` kindly fix the `z-index` 
### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/157667823-b5dc7606-c1c0-4764-b9b5-aa96e3d6617f.png)

### Figma's Design
![image](https://user-images.githubusercontent.com/59416136/157667718-cc360e25-26ee-43de-a276-e8cb3d5657c4.png)